n26/core/templates/cotton/n26/profile_picker/row.html

```
@@ -229,29 +229,36 @@
+    without a card_url carries its drawn card inline — the gallery's
+    sample rows have no endpoint to fetch from.
     {% endcomment %}
     <c-slot name="details">
         {% for option in entry.options %}
-            {% if forloop.first %}
-                <div x-show="mainpick === {{ forloop.counter0 }}">
-                    <c-n26.model-card :card="option.card" />
-                </div>
-            {% else %}
-                <div x-show="mainpick === {{ forloop.counter0 }}" x-cloak>
+            <div x-show="mainpick === {{ forloop.counter0 }}"{% if not forloop.first %} x-cloak{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)